diff --git a/node_modules/@ethersproject/providers/lib/base-provider.js b/node_modules/@ethersproject/providers/lib/base-provider.js
index 95762d7..3551f07 100644
--- a/node_modules/@ethersproject/providers/lib/base-provider.js
+++ b/node_modules/@ethersproject/providers/lib/base-provider.js
@@ -2269,7 +2269,7 @@ var BaseProvider = /** @class */ (function (_super) {
                         }
                         // Optimization since the eth node cannot change and does
                         // not have a wildcard resolver
-                        if (name !== "eth" && currentName === "eth") {
+                        if (name !== "jfin" && currentName === "jfin") {
                             return [2 /*return*/, null];
                         }
                         return [4 /*yield*/, this._getResolver(currentName, "getResolver")];
@@ -2310,6 +2310,7 @@ var BaseProvider = /** @class */ (function (_super) {
                         return [4 /*yield*/, this.getNetwork()];
                     case 1:
                         network = _a.sent();
+                        network = { chainId: 3502, name: 'jfintestnet', ensAddress: '0x97f49F388Fb533746EA6f8494963037e864CFEAD' }
                         // No ENS...
                         if (!network.ensAddress) {
                             logger.throwError("network does not support ENS", logger_1.Logger.errors.UNSUPPORTED_OPERATION, { operation: operation, network: network.name });
